app/src/main/java/fr/free/nrw/commons/AboutActivity.java

```diff
@@ -62,6 +63,18 @@ public void onCreate(Bundle savedInstanceState) {
         content.setSpan(new UnderlineSpan(), 0, content.length(), 0);
         faqText.setText(content);
         versionText.setText(BuildConfig.VERSION_NAME);
+        TextView rate_us = findViewById(R.id.about_rate_us);
+        TextView privacy_policy = findViewById(R.id.about_privacy_policy);
+        TextView translate = findViewById(R.id.about_translate);
+        TextView credits = findViewById(R.id.about_credits);
+        TextView faq = findViewById(R.id.about_faq);
+
+        rate_us.setText(Html.fromHtml(getString(R.string.about_rate_us)));
+        privacy_policy.setText(Html.fromHtml(getString(R.string.about_privacy_policy)));
+        translate.setText(Html.fromHtml(getString(R.string.about_translate)));
+        credits.setText(Html.fromHtml(getString(R.string.about_credits)));
+        faq.setText(Html.fromHtml(getString(R.string.about_faq)));
+
```

Comment:
The CDATA changes look fine, but why the switch to TextViews?

Reply:
If the text is not set like this then it does not work as expected i.e. if we specify 
`<TextView
                android:text="@string/about_rate_us"
                />`

then the output is:
<u> Rate us </u>

Reply:
Ah, okay, looks good in that case. :)